react-calculator/src/App.js

```diff
@@ -0,0 +1,169 @@
+import "./App.css";
+import React, { Component } from "react";
+import Digit from "./component/Digit";
+import Operation from "./component/Operation";
+import {
+  checkMaxNumberLength,
+  checkValidEqualOperation,
+  checkValidOperation,
+} from "./util/validator.js";
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      numbers: ["", ""],
+      operator: "",
+      calculated: false,
+    };
```

Comment:
```suggestion
    this.state = initialState
```
리셋에서도 사용하는 만큼 변수로 분리해두는것도 좋을거같아요